tests/ui/coherence/orphan-check-projections-dont-cover.classic.stderr

```diff
@@ -0,0 +1,21 @@
+error[E0210]: type parameter `T` must be covered by another type when it appears before the first local type (`Local`)
+  --> $DIR/orphan-check-projections-dont-cover.rs:20:6
+   |
+LL | impl<T> foreign::Trait<Local, T, ()> for <T as Identity>::Output {}
+   |      ^ type parameter `T` must be covered by another type when it appears before the first local type (`Local`)
+   |
+   = note: implementing a foreign trait is only possible if at least one of the types for which it is implemented is local, and no uncovered type parameters appear before that first local type
```

Comment:
While I could update the diagnostic to say something like
```suggestion
   = note: implementing a foreign trait is only possible if at least one of the types for which it is implemented is local, and no type parameters not covered by a nominal type appear before that first local type
```

we could instead change the definition of "covered" (e.g., in the Rust Reference) to reflect that the covering type has to be nominal.